FPGA1212_FLAT_HD_SKY_PNR/FPGA1212_FLAT_HD_SKY_task/arch/vpr_arch.xml

```diff
@@ -396,18 +396,21 @@
               <direct name="direct3" input="ff[0].Q" output="ff[1].DI"/>
               <direct name="direct4" input="ff[1].Q" output="fabric.sc_out"/>
               <direct name="direct5" input="ff[1].Q" output="fabric.reg_out"/>
-              <direct name="direct6" input="frac_logic.out[1:1]" output="ff[1:1].D"/>
               <complete name="complete1" input="fabric.clk" output="ff[1:0].clk"/>
               <mux name="mux1" input="frac_logic.out[0:0] fabric.reg_in" output="ff[0:0].D">
                 <delay_constant max="25e-12" in_port="frac_logic.out[0:0]" out_port="ff[0:0].D"/>
                 <delay_constant max="45e-12" in_port="fabric.reg_in" out_port="ff[0:0].D"/>
               </mux>
-              <mux name="mux2" input="ff[0].Q frac_logic.out[0]" output="fabric.out[0]">
+              <mux name="mux2" input="frac_logic.out[1:1] ff[0:0].Q" output="ff[1:1].D">
```

Comment:
Are you using this architecture in the PnR? It is the patched version.

Reply:
Yes, using the most updated architecture. You can check ``**_task/arch/`` directory for the exact file

Reply:
Just checked your [post PnR netlist](https://raw.githubusercontent.com/LNIS-Projects/skywater-openfpga/66d09da857912fd2810316b38acd9010258408a2/FPGA1212_FLAT_HD_SKY_PNR/fpga_top/fpga_top_icv_in_design.pt.v).
It is indeed the latest architecture. Good work.